observation/src/main/java/com/linecorp/armeria/client/observation/HttpClientObservationConvention.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client.observation;
+
+import io.micrometer.observation.Observation.Context;
+import io.micrometer.observation.ObservationConvention;
+
+interface HttpClientObservationConvention extends ObservationConvention<HttpClientContext> {
```

Comment:
Question) Is it possible to remove this interface and just add `supportContext` directly into `DefaultHttpClientObservationConvention`?